creator/decorators.py

```diff
@@ -181,12 +175,7 @@ def close(self):
         if len(job_logs) == 0:
             job_logs = [JobLog(job=self._job, error=self._job.failing)]
 
-        self.logger.info(
-            f"Job complete after {duration:.2f}s. "
-            f"Saving to Job Logs: "
-            f"{', '.join(yellow(str(log.id)) for log in job_logs)}, "
-            f"goodbye! 👋"
-        )
+        self.logger.info(f"Job complete after {duration:.2f}s. 👋")
```

Comment:
I liked part that logged the ID of the job log. Can we add that into this line or make a new line for it?